hat/assets/js/apps/Iaso/domains/tasks/components/NotificationImportDetailModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FunctionComponent } from 'react';
+
+import {
+    LoadingSpinner,
+    makeFullModal,
+    useSafeIntl,
+} from 'bluesquare-components';
+import { AlertModal } from '../../../components/AlertModal/AlertModal';
+
+import MESSAGES from '../messages';
+import { NotificationImportDetailButton } from './NotificationImportDetailButton';
+import { Task } from '../types';
+import { useGetPolioNotificationImport } from '../hooks/api';
+
+type Props = {
+    task: Task<any>;
+    isOpen: boolean;
+    closeDialog: () => void;
+};
+
+const NotificationImportDetailModal: FunctionComponent<Props> = ({
+    task,
+    isOpen,
+    closeDialog,
+}) => {
+    const { formatMessage } = useSafeIntl();
+
+    const {
+        data: notificationImport,
+        isFetching: isFetchingNotificationImport,
+    } = useGetPolioNotificationImport(task.polio_notification_import_id);
+
+    return (
+        <AlertModal
+            isOpen={isOpen}
+            closeDialog={closeDialog}
+            titleMessage={formatMessage(
+                MESSAGES.polioNotificationImportDetails,
+            )}
+            id="polio-notifications-import-details-modal"
+            maxWidth="md"
+        >
+            {isFetchingNotificationImport && <LoadingSpinner />}
+            {!isFetchingNotificationImport && (
+                <>
+                    <p>{task.result.message}</p>
+                    {notificationImport &&
+                        notificationImport.errors.length > 0 && (
+                            <>
+                                <p>
+                                    {formatMessage(
+                                        MESSAGES.polioNotificationImportErrors,
+                                        {
+                                            count: notificationImport.errors
+                                                .length,
+                                        },
+                                    )}
+                                </p>
+                                <pre style={{ maxHeight: '550px' }}>
```

Comment:
I don't know if it should be like that, but I checked the [mockup](https://whimsical.com/polio-line-list-import-error-check-DDJfGsn9n4yMpsJvo19KCY) and find that the error data or updated ones should not appear like json objects but instead like a list.

Reply:
Yes, because the solution in the mockup would have required a too complex implementation.

So I suggested using a JSON field to store the errors.